templates/org/home.tmpl

```diff
@@ -15,7 +15,7 @@
 
 			{{if .ShowMemberAndTeamTab}}
 			<div class="ui five wide column">
-				{{if .CanCreateOrgRepo}}
+				{{if and .CanCreateOrgRepo .SignedUser.CanCreateRepo}}
```

Comment:
Whether `.SignedUser` will not be `nil`?

Reply:
In org, it seems that it is safe.
![image](https://github.com/user-attachments/assets/209307fb-1e73-4378-8a73-06eedd78a6f2)